tests/queries/0_stateless/01546_log_queries_min_query_duration_ms.sql

```diff
@@ -0,0 +1,10 @@
+set log_queries=1;
+set log_queries_min_query_duration_ms=1;
+
+select '01546_log_queries_min_query_duration_ms-fast', * from system.one format Null;
+system flush logs;
+select count() from system.query_log where query like '%01546_log_queries_min_query_duration_ms-fast%' and query not like '%system.query_log%' and event_date = today() and event_time >= now() - interval 1 minute;
+
+select '01546_log_queries_min_query_duration_ms-slow', sleep(2) from system.one format Null;
+system flush logs;
+select count() from system.query_log where query like '%01546_log_queries_min_query_duration_ms-slow%' and query not like '%system.query_log%' and event_date = today() and event_time >= now() - interval 1 minute;
```

Comment:
count() > 0 will solve the issue with iterated runs.

Reply:
>count() > 0 will solve the issue with iterated runs.

But this can hide problems on the local runs, during development/debugging, and I was trying to avoid such things.

Here is an idea using remote() to resolve the currentDatabase() into the database name, and since clickhouse-test runs each test in a separate database this works:

```
create table one engine=Memory() as select * from system.one;

set log_queries_min_time=1;
set log_queries=1;

-- this is required to have query with the currentDatabase() resolved
set prefer_localhost_replica=0;
select '01546_log_queries_min_query_duration_ms-slow', sleep(2) from remote('127.1', currentDatabase(), one) format Null;
system flush logs;
select count()
from system.query_log
where
    query like '%01546_log_queries_min_query_duration_ms-slow%'
    and query like concat('% FROM ', currentDatabase(), '.one')
    and query not like '%system.query_log%'
    and event_date = today()
    and event_time >= now() - interval 1 minute;
```

Reply:
That's a good idea.
PS. Maybe better to add `current_database` field to `query_log` and `query_thread_log`?

Reply:
>PS. Maybe better to add current_database field to query_log and query_thread_log?

Yes, this will make lots of tests (that uses query_log/query_thread_log) simpler.
Ok, This should be done separately.

Reply:
>That's a good idea.

Ok, will apply and squash the commits